nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -233,9 +233,9 @@ def test_find_parcellation_cut_coords():
     x_map_b, y_map_b, z_map_b = (30, 30, 30)
     x_map_c, y_map_c, z_map_c = (50, 50, 50)
     # Defining 3 parcellations
-    data[x_map_a - 10:x_map_a + 10, y_map_a - 10:y_map_a + 10, z_map_a - 10: z_map_a + 10] = 1
-    data[x_map_b - 10:x_map_b + 10, y_map_b - 10:y_map_b + 10, z_map_b - 10: z_map_b + 10] = 2
-    data[x_map_c - 10:x_map_c + 10, y_map_c - 10:y_map_c + 10, z_map_c - 10: z_map_c + 10] = 3
+    data[x_map_a - 10:x_map_a + 10, y_map_a - 10:y_map_a + 10, z_map_a - 10: z_map_a + 10] = 2301
+    data[x_map_b - 10:x_map_b + 10, y_map_b - 10:y_map_b + 10, z_map_b - 10: z_map_b + 10] = 4001
```

Comment:
It looks like PEP8 is complaining about these lines being too long (they were probably added before we decided to enforce PEP8 conventions).
Could you break them?

Reply:
I broke the lines and add the fixed contents in doc/whats_new